meteoritics-and-planetary-science.csl

```diff
@@ -125,8 +129,8 @@
   </citation>
   <bibliography hanging-indent="true" et-al-min="10" et-al-use-first="1">
     <sort>
-      <key macro="author"/>
-      <key variable="title"/>
+      <key macro="author-short" names-use-first="1" names-min="2"/>
+      <key macro="year-date"/>
```

Comment:
sorry, I'm confused now. Rintze and I were talking about the bibliography, but your code is for the in-text citation.  Also, your code now is always chronological, which I don't think you want. I.e. it would do (Zang 2000; Kita 2005)